docassemble/AssemblyLine/al_general.py

```diff
@@ -479,12 +480,30 @@ def signature_if_final(self, i: str) -> Union[DAFile, str]:
         else:
             return ""
 
-    def phone_numbers(self) -> str:
+    def phone_numbers(self, country=None) -> str:
         nums = []
         if hasattr(self, "mobile_number") and self.mobile_number:
-            nums.append({self.mobile_number: "cell"})
+            try:
+                nums.append(
+                    {
+                        phone_number_formatted(
+                            self.mobile_number, country=country
+                        ): "cell"
+                    }
+                )
+            except:
```

Comment:
Did you run into an exceptions when doing this, or are you just being extra cautious? Looking through `phone_number_formatted`, it doesn't look like there's anything that would throw, I'm okay with either, just curious.

Reply:
I think the function used to return an exception if the phone number didn't match the country code but it looks like it doesn't anymore. So might be redundant now.